workers/unity/Packages/com.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -77,90 +76,108 @@ private Worker(string workerType, Connection connection, ILogDispatcher logDispa
         }
 
         /// <summary>
-        ///     Asynchronously connects and creates a worker via the Receptionist.
+        ///     Tries to connect to the SpatialOS Runtime and creates the worker responsible for the connection upon successfully connecting.
         /// </summary>
-        /// <param name="config">The Receptionist connection configuration.</param>
-        /// <param name="logger">The logger for this worker.</param>
-        /// <param name="origin">The origin of this worker in local Unity space.</param>
-        /// <returns>A task that returns a Worker when finished.</returns>
-        /// <exception cref="ConnectionFailedException">
-        ///     Thrown if the worker fails to connect.
-        /// </exception>
-        public static async Task<Worker> CreateWorkerAsync(ReceptionistConfig config, ILogDispatcher logger,
+        /// <param name="connectionFuture">The <see cref="Future{T}"/> of the <see cref="Connection"/> object that we use to connect to the SpatialOS Runtime.</param>
+        /// <param name="workerType">The type of the worker.</param>
+        /// <param name="logger">The logger used by the worker.</param>
+        /// <param name="origin">The position of the worker.</param>
```

Comment:
Why was the text changed from "the origin of this worker in local Unity space" changed to "The position of the worker"? The latter is more ambiguous